pyramid_oereb/standard/models/main.py

```diff
@@ -192,6 +192,23 @@ class ExclusionOfLiability(Base):
     content = sa.Column(JSONType, nullable=False)
 
 
+class LawStatus(Base):
+    """
+    The container where you can throw in all the law status texts this application
+    should have access to, for creating extracts.
+    Attributes:
+        id (int): The identifier. This is used in the database only and must not be set manually. If
```

Comment:
do we need the id? Should the code not be a primary key already?

Reply:
I removed the Id already. Good issue ;)